airbyte-webapp/src/utils/cron/validateCronExpression.test.ts

```diff
@@ -63,7 +63,21 @@ describe("validateCronExpression", () => {
     ${"wildly invalid"}                                      | ${false}
     ${"* * * * *"}                                           | ${false}
     ${"0 0 0 0 0 0"}                                         | ${false}
-  `("'$expression' is valid: $isValid", async ({ expression, isValid }) => {
+  `("'$expression' is valid: $isValid", ({ expression, isValid }) => {
     expect(validateCronExpression(expression)).toEqual(isValid);
   });
 });
+
+describe("validateCronFrequencyOverOneHour", () => {
+  it.each`
+    expression            | isValid
+    ${"0 0 12 * * ?"}     | ${true}
+    ${"0 0 * * * ?"}      | ${true}
+    ${"0 * 12 * * ?"}     | ${false}
+    ${"* * 12 * * ?"}     | ${false}
+    ${"15,45 * 12 * * ?"} | ${false}
+    ${"0 15,45 12 * * ?"} | ${false}
```

Comment:
totally optional suggestion: add an interval case, like
```
    ${"0 0/10 * * * ?} | ${false}
```
for "every ten minutes"

Reply:
Good call! I also took the opportunity to split the fields by `/\s+/g` instead of `" "`, since any number of spaces can be used to delimit the cron fields.